rbac/internal/views.py

```diff
@@ -545,6 +546,44 @@ def list_bindings_for_role(request):
     return HttpResponse(json.dumps(result), content_type="application/json", status=200)
 
 
+def migration_resources(request):
+    """View or delete specific resources related to migration.
+
+    DELETE /_private/api/utils/migration_resources/?resource=xxx&org_id=xxx
+    GET /_private/api/utils/migration_resources/?resource=xxx&org_id=xxx&limit=1000
+    optinos of resource: workspace, mapping(tenantmapping), binding(bindingmapping)
```

Comment:
typo `optinos`

Reply:
Oops, forgot to update this, good catch!